src/user-guide/cheat-sheet.rst

```diff
@@ -0,0 +1,292 @@
+resume a workflow::
+
+   # pause a workflow
+   cylc pause <id>
+
+   # resume a workflow
+   cylc play <id>
+
+The Cylc play, pause and stop commands work similarly to the play, pause and
+stop buttons on an old tape player. You can play, pause or stop a workflow as
+many times as you like, Cylc will never loose its place in the workflow.
+
+
+List Workflows
+--------------
+
+The ``cylc scan`` command can list your installed workflows::
+
+   # list all running workflows
+   cylc scan
+
+   # get information about running workflows
+   cylc scan -t rich
+
+   # list all installed workflows
+   cylc scan --states=all
+
+You can also view your workflows in the GUI or Tui.
+
+
+Opening the GUI/Tui
+-------------------
+
+Cylc has an in-terminal utility for monitoring and controlling workflows::
+
+   # view all workflows
+   cylc tui
+
+   # open a specific workflow
+   cylc tui <id>
+
+There is also a GUI which opens in a web browser::
+
+   # open the GUI to the homepage
+   cylc gui
+
+   # open the GUI to a specific workflow
+   cylc gui <id>
+
+
+Making Changes To Running Workflows
+-----------------------------------
+
+You can make changes to a workflow without having to shut it down and restart it.
+
+First, make your required changes to the files in the workflow's
+:term:`source directory`, then run the ``cylc vr`` command
+(:ref:`more information <interventions.edit-the-workflow-configuration>`)::
+
+   # validate, reinstall and reload the workflow
+   cylc vr <id>
+
+If you want to quickly edit a task's configuration, e.g. whilst developing a
+workflow or testing changes, the 
+:ref:`"Edit Runtime" feature <interventions.edit-a-tasks-configuration>`
+in the GUI can be convenient.
+
+
+Inspecting Workflows
+--------------------
+
+Validate the workflow configuration (good for spotting errors)::
+
+   cylc validate <path/id>
+
+View the workflow configuration *before* Cylc has parsed it
+(good for debugging Jinja2 errors)::
+
+   cylc view -p <path/id>
+
+View the workflow configuration *after* Cylc has parsed it
+(good for debugging family inheritance)::
+
+   cylc config <path/id>
+
+   # view a specific task's configuration
+   cylc config <path/id> -i '[runtime][<task>]'
+
+   # view the workflow configuration with defaults applied
+   cylc config <path/id> --defaults
+
+Generate a graphical representation of the workflow's :term:`graph`
+(a useful tool for developing workflow graphs)::
+
+   cylc graph <path/id>
+
+   # render the graph between two cycle points
+   cylc graph <path/id> <cycle1> <cycle2>
+
+   # render the graph transposed (can make it easier to read)
+   cylc graph <path/id> --transpose
+
+   # group tasks by cycle point
+   cylc graph <path/id> --cycles
+
+   # collapse tasks within a family (can reduce the number of tasks displayed)
+   cylc graph <path/id> --group=<family>
+
+List all tasks and families defined in a workflow::
+
+   cylc list <path/id>
+
```

Comment:
Might be a place to refer to `cylc lint`?

Reply:
Done